CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Patches to Waterfall are very simple, but center around the directory 'Waterfall
 Assuming you already have forked the repository:
 
 1. Pull the latest changes from the main repository
-2. Type `git submodule update --init && ./applyPatches.sh` in git bash to apply the changes from upstream
+2. Type `git submodule update --init && ./scripts/applyPatches.sh` in git bash to apply the changes from upstream
```

Comment:
This needs to be `./waterfall p`, the script also runs the submodule update command, so that can be removed too :)